scripts/cron.py

```diff
@@ -70,6 +58,15 @@ def job_update_product_details():
     connections.close_all()
 
 
+@scheduled_job('cron', month='*', day='*', hour='*', minute='*', max_instances=1, coalesce=True)
+@babis.decorator(ping_after=settings.DEAD_MANS_SNITCH_DISABLE_SNIPPETS)
+def job_disable_snippets_past_publish_date():
+    call_command('disable_snippets_past_publish_date')
+    # Django won't close db connections after call_command. Close them manually
```

Comment:
Just a note: turns out a lot of these management commands assume they're not in a long-running process, so when doing this in bedrock I just [shell out and call the management command as a separate process](https://github.com/mozilla/bedrock/blob/d26bad24956a107a76ff5da6bb9297c6d1747457/bin/cron.py#L25-L26). It's not that much more time and avoids some issues like this. Not an r-, just something to keep in mind if more strange behavior is found.

Reply:
Actually this is good advice and I'm going to take it.